### PR TITLE
Remove unnecessary webpack process.env plugin push

### LINKED ISSUE #1816: 🐛Environmental variable references no longer working
**:bug: Describe the bug**
A previous change : https://github.com/equinor/witsml-explorer/pull/1735
Added "NODE_ENV" prefix to the variable name when pushing environmental variables to the build output.
e.g. WITSMLEXPLORER_API_URL->NODE_ENVWITSMLEXPLORER_API_URL
This resulted in the current references through process.env, e.g.  process.env.WITSMLEXPLORER_API_URL became ineffective.
(Because it has changed to process.env.NODE_ENVWITSMLEXPLORER_API_URL)
**To Reproduce**
Add WITSMLEXPLORER_FRONTEND_URL value and WITSMLEXPLORER_API_URL in .env file

**Expected behavior**
The returned base path of AssetLoader and the backend url should be modified

**Screenshots**
The base path of AssetLoader and the backend url are remain unchanged

### PR DESCRIPTION
**Pull request by kongxy10**
## Fixes

This pull request fixes #1816

## Description
In current next.config.js implementation, we are pushing the variables defined in .env files into process.env, and add a NODE_ENV prefix to the variable name references. 
This step is no longer needed now, since the variables with NEXT_PUBLIC_ prefix will directly available in browser runtime with process.env.NEXT_PUBLIC....
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

If we keep the plugin pushing code block, we will eventually got 2 duplicate references for each variable defined in .env file.
For example: process.env.NEXT_PUBLIC_AZURE_AD_TENANT_ID and process.env.NODE_ENVNEXT_PUBLIC_AZURE_AD_TENANT_ID for the same variable NEXT_PUBLIC_AZURE_AD_TENANT_ID

## Type of change

[//]: # (Mark any of the types of change that apply.)

* [ ] Bugfix
* [ ] New feature (non-breaking change which adds functionality)
* [x] Enhancement of existing functionality
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## Impacted Areas in Application

[//]: # (List general components of the application that this PR will affect)

* [x] Fronted
* [ ] API
* [ ] WITSML
* [ ] Other (please describe)

## Checklist:

[//]: # (Please tick all the boxes or remove the ones that aren't needed and explain why)

*Communication*
* [ ] I have made corresponding changes to the documentation
* [ ] PR affects application security

*Code quality*
* [x] I have self-reviewed my code
* [ ] No new warnings are generated

*Test coverage*
* [ ] New code is covered by passing tests

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: Src/WitsmlExplorer.Frontend/next.config.js
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/next.config.js]: `// eslint-disable-next-line @typescript-eslint/no-var-requires` can now be removed as well

Comment (kongxy10) [reply Src/WitsmlExplorer.Frontend/next.config.js]: removed

**Review by janburak — APPROVED** [review]
- [X] Read issue and PR
- [X] Pulled branch and manually tested
- [X] Verify that PR resolves issue
- [X] Reviewed the code

**Comment by janburak** [comment]
@kongxy10 Do you have the permission the merge PRs once they are approved, or should I merge the PR?

**Comment by kongxy10** [comment]
@janburak Please help to merge for me. I don't have write access.

**Pull request merged**
